ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/js/signin.js

```diff
@@ -1,10 +1,15 @@
-	const findRedriectUrl = function(domId) {
+let hasCart = false;
+const findRedriectUrl = function(domId) {
 		let redirectURL = "";
 		const params = new URLSearchParams(window.location.search);
 		const keys = ['signin.html', 'signup.html'];
 		if(params.has('referer')) {
 			if(!keys.some(k => params.get('referer').includes(k) )) {
 				redirectURL = params.get('referer');
+                if(!hasCart && redirectURL.includes('checkout.html'))
+                {
+					redirectURL = "/content/hclecomm/us/en/home.html";
```

Comment:
@DeepaliAgrwl : The path should not be hard coded . Please author default redirect path in sign in component

Reply:
Hard coded URL has been replaced by the default url authored in the component.